### PR TITLE
Custom auth provider

### PR DESCRIPTION
**Pull request by gyorgykt**
This update helps you configure the credential store to use a custom provider.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 requests_toolbelt = "*"
+yarl = "*"
 
 [dev-packages]
 oneidentity-safeguard-sessions-plugin-sdk = "*"
 
 [requires]
-python_version = "3.6"
-
+python_version = "3.10"
```
Comment (OneIdentityPAMIntegration) [inline Pipfile:9]: This should match the version of python on SPS

**Review by OneIdentityPAMIntegration — CHANGES_REQUESTED** [review]

File: MANIFEST [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-version: 1.0.0
+version: 1.1.0
 type: aa
-api: 1.6
+api: 1.7.0
 entry_point: main.py
 author_name: One Identity Safeguard Remote Access
 author_email: QDL.QBU-OI.RD.Safeguard.Integration@oneidentity.com
-scb_max_version: 6.13.0~
+scb_max_version: 8.0.0~
```
[the remark this answers is not in the document]
Comment (gorosz) [reply MANIFEST:9]: Same as above

File: Pipfile (hunk 1 above)
Comment (gorosz) [reply Pipfile:9]: Same as above

File: plugin/plugin.py [hunk 3]
```diff
@@ -98,6 +102,7 @@ def do_authenticate(self):
 
     # SPS initiated code path should be extracted. pylint: disable=too-many-return-statements
     def do_authorize(self):
+        self.session_cookie.setdefault("SessionId", self.connection.session_id)
```
[the remark this answers is not in the document]
Comment (gorosz) [reply plugin/plugin.py:105]: Same as above

**Review by gorosz — CHANGES_REQUESTED** [review]

File: MANIFEST (hunk 2 above)
[the remark this answers is not in the document]
Comment (gyorgykt) [reply MANIFEST:9]: 1. Nobody tell me we don't want...
2. You may find a release where there is no version limit. It caused the error.

My other opinion was SPS 7.5.

File: Pipfile (hunk 1 above)
Comment (gyorgykt) [reply Pipfile:9]: on SPS 7.4.0 the python version is 3.10.x
so is it wrong?

File: plugin/plugin.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (gyorgykt) [reply plugin/plugin.py:105]: https://github.com/OneIdentity/safeguard-sessions-plugin-sra-initiated/issues/3#issue-1802999541

**Review by OneIdentityPAMIntegration — APPROVED** [review]

**Review by gorosz — APPROVED** [review]

**Pull request merged**